mesh/silvair-io.c

```diff
@@ -715,19 +712,18 @@ struct silvair_io *silvair_io_new(int fd,
 void silvair_io_keep_alive_wdt_refresh(struct silvair_io *io)
 {
 	if (!io)
-		return;
-
-	if (!io->keep_alive_watchdog)
-		return;
+		abort();
 
 	if (io->disconnect_watchdog) {
 		l_timeout_remove(io->disconnect_watchdog);
 		io->disconnect_watchdog = NULL;
 		l_info("Connection OK");
 	}
 
-	l_timeout_modify_ms(io->keep_alive_watchdog,
-					keep_alive_watchdog_period_ms);
+	if (io->keep_alive_watchdog)
```

Comment:
I don't think this can ever happen, `keep_alive_watchdog` is not set to `NULL` anywhere after this change.